manifests/prometheus-serviceMonitorKubelet.yaml

```diff
@@ -18,6 +18,19 @@ spec:
     scheme: https
     tlsConfig:
       insecureSkipVerify: true
+ - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
```

Comment:
This file is generated from jsonnet code in `jsonnet/`. CI fails because this is removed when running generation.